docs/src/app/@theme/components/header/header.component.ts

```diff
@@ -17,6 +17,7 @@ import { NgdVersionService } from '../../services';
     </div>
     <div class="section middle">
       <nb-menu [items]="mainMenu"></nb-menu>
+      <ngd-search *ngIf="showSearch"></ngd-search>
```

Comment:
`showSearch` is never changed. You can remove `ngIf` statement as well as `showSearch` class property.